main.css

```diff
@@ -0,0 +1,591 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: var(--font);
+  background-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.navbar {
+  display: flex;
+  width: 375px;
+  height: 48px;
+  padding: 4px 24px 4px 12px;
+  align-items: flex-start;
+  gap: 190px;
+  flex-shrink: 0;
+}
+
+.navbar__logo {
+  font-size: 18px;
+  margin-left: 30px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 20px;
+  color: #6070ff;
+  padding-top: 40px;
+  font-family: Poppins, sans-serif;
```

Comment:
- [ ] [OPTIONAL] You can declare Poppins font in your body tag so that all your other CSS tags will inherit from it, then you won't need to repeat the same code all over your CSS file. This will help keep your code dry.